integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/bootstrap/ElasticsearchBootstrapIT.java

```diff
@@ -127,7 +127,11 @@ public void noVersionCheck_incompleteVersion() {
 						" use the same model dialect." +
 						" It is not the case on ES 5 in particular, since 5.6 has a dialect" +
 						" but 5.0, 5.1, etc. don't have one.",
-				isAtMost( ElasticsearchTestDialect.getActualVersion(), "elastic:5.6" )
+				condition(
+						ElasticsearchTestDialect.getActualVersion(),
+						esVersion -> esVersion.isAtMost( "5.6" ),
+						osVersion -> osVersion.none()
```

Comment:
That's... unnecessarily complex? Personally I think a simple `false` here would make much more sense.